tsconfig.json

```diff
@@ -2,8 +2,9 @@
   "compilerOptions": {
     "strict": true,
     "target": "ESNext",
-    "moduleResolution": "Node",
+    "moduleResolution": "node",
```

Comment:
just curious, does this matter, no right?

Reply:
I didn't actually see a difference here no, I just did it for consistency as I was trying to figure out some other config changes